src/_P001_Switch.ino

```diff
@@ -324,7 +364,10 @@ boolean Plugin_001(byte function, struct EventStruct *event, String& string)
               outputstate[event->TaskIndex] = new_outputState;
               boolean sendState = new_outputState;
 
-              if (Settings.TaskDevicePluginConfig[event->TaskIndex][7]==3 && Settings.TaskDevicePluginConfig[event->TaskIndex][4])
+              if (Settings.TaskDevicePin1Inversed[event->TaskIndex])
+                sendState = !sendState;
+
+              if (Settings.TaskDevicePluginConfig[event->TaskIndex][7]==3 && Settings.TaskDevicePluginConfig[event->TaskIndex][4]>0)
```

Comment:
Those `COUNTER` and `DC` defines may be extended to here also (maybe the whole plugin?)
And to avoid future redefine issues, maybe prefix the name with the plugin number?

It makes the code a lot more readable and maybe we should do it on more plugins.

Reply:
The idea of DC and COUNTER were only to simplify the "if" clause.
I could change them to P001_DC and P001_COUNTER and make the change in all 3 plugin for all the settings.
In order to schedule the work, is this the only change requested?

Reply:
Yep, if nobody objects, I will merge it. :)